.circleci/config.yml

```diff
@@ -271,6 +271,36 @@ jobs:
           when: always
       - store_artifacts:
           path: /tmp/cni-test
+  custom_networking_test:
```

Comment:
Why not use GitHub Actions workflows instead?

We plan to remove Circle CI and moved everything to GitHub Actions. Plus this will leverage the self hosted runners we added

Reply:
Yes Fawad, will move it to GitHub actions.